torch_frame/data/stats.py

```diff
@@ -85,22 +85,25 @@ def compute(
         sep: str | None = None,
     ) -> Any:
         if self == StatType.MEAN:
-            flattened = np.hstack(np.hstack(ser.values))
+            val = np.hstack(ser.values) if ser.values.ndim > 1 else ser.values
+            flattened = np.hstack(val) if val.ndim > 1 else val
```

Comment:
Looks like the chagne doesn't take account of the `sequence_numerical` case:
```
self = <StatType.MEAN: 'MEAN'>
ser = 0                                                 [nan]
1         [0.3309801272693331, nan, 0.3018088119575514]
2     ...42, nan, 0.9997390177887806, ...
19                            [nan, 0.36[1182](https://github.com/pyg-team/pytorch-frame/pull/449/checks#step:6:1183)4486676457]
Name: seq_num_1, dtype: object
sep = None

    def compute(
        self,
        ser: Series,
        sep: str | None = None,
    ) -> Any:
        if self == StatType.MEAN:
            val = np.hstack(ser.values) if ser.values.ndim > 1 else ser.values
            flattened = np.hstack(val) if val.ndim > 1 else val
>           finite_mask = np.isfinite(flattened)
E           TypeError: ufunc 'isfinite' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe''

torch_frame/data/stats.py:90: TypeError
```
https://github.com/pyg-team/pytorch-frame/actions/runs/12376953024/job/34545299882